### PR TITLE
[PKG-7402] API docs for registries index

### PR DESCRIPTION
**Pull request by ensvo**
New section for Packages API endpoints - Registries.

image

### CODE REVIEW

File: data/nav.yml [hunk 1]
```diff
@@ -557,7 +557,11 @@
             - name: "Members"
               path: "apis/rest-api/organizations/members"
               pill: "beta"
-        - name: "Pipelines "
```
Comment (ensvo) [inline data/nav.yml:560]: Removed extra space.

**Review by swebb — APPROVED** [review]
👍🏻

File: data/nav.yml (hunk 1 above)
Comment (gilesgas) [reply data/nav.yml:560]: No this needs to be there unfortunately for the reason I've now explained in a comment within the relevant areas of the `nav.yml` file.

**Review by gilesgas — APPROVED** [review]
LGTM now! I'll review and merge the others

**Pull request merged**